### ISSUE TITLE
[Bug]:  Not Work Buttons Q/E at take server battery in MiSide

### ISSUE THREAD

**Issue by MrSliTraP2716Lua** [originalnicodr/CinematicUnityExplorer#98]
### Are you on the latest version of CinematicUnityExplorer?

- [X] Yes, I'm on the latest version of CinematicUnityExplorer.

### Which release are you using?

BepInEx IL2CPP

### Which game did this occur on?

[output_log.txt](https://github.com/user-attachments/files/18276233/output_log.txt)
[LogOutPut.log](https://github.com/user-attachments/files/18276235/LogOutPut.log)

### Describe the issue.

Version: 1.3.0
Not work buttons Q/E  at take server battery in game MiSide 0.9, 0.9.1f. 0.9.2.1 & Demo versions (Demo on Mono & Demo on Il2CPP)

**Comment by holetoo233** [comment]
I also encountered the same issue, have you resolved it?

**Comment by originalnicodr** [comment]
Hey, have you tried to remap the camera tilt buttons on the CinematicUnityExplorer options menu by any chance? (last tab on the navbar) If not please try that and report back.

**Comment by holetoo233** [comment]
> Hey, have you tried to remap the camera tilt buttons on the CinematicUnityExplorer options menu by any chance? (last tab on the navbar) If not please try that and report back.

Are you referring to the key bindings for "Tilt left," "Tilt right," and "Tilt reset" in the options menu? I see their defaults are keypad1, keypad3, and keypad2. None of the other keys in the options conflict with the in-game interaction key "e." Should I try changing these keys to the e key?

**Comment by originalnicodr** [comment]
My bad, for some reason I thought that these keys were assigned to Q and E.

I imagine this is not happening while on freecam (as the mod tries to block input to the game) but rather on normal gameplay, is that right? Could you try unticking the "Block games input on Freecam" on the freecam panel and see if that fixes it?

It is a weird bug, could try the demo out and check what is going on myself, but can't confirm when I will get to it or if I will be able to fix it.

**Comment by holetoo233** [comment]
> My bad, for some reason I thought that these keys were assigned to Q and E.
> 
> I imagine this is not happening while on freecam (as the mod tries to block input to the game) but rather on normal gameplay, is that right? Could you try unticking the "Block games input on Freecam" on the freecam panel and see if that fixes it?
> 
> It is a weird bug, could try the demo out and check what is going on myself, but can't confirm when I will get to it or if I will be able to fix it.

Yes, I have tried both the normal game perspective and the free mode perspective, but neither can normally trigger any interactions with "e" and "q" in this game. I have attempted to uncheck the option "Block games input on Freecam" on the free panel, and also tried changing "Disable EventSystem override = false" in "com.originalnicodr.cinematicunityexplorer" under the directory "beplnEx/config". I have tried changing it to true, but none of these methods allow me to proceed with the story normally.
Thank you for your reply!

**Comment by MrSliTraP2716Lua** [comment]
The problem is solved if you install the fork version of Unity Explorer 4.1

**Comment by originalnicodr** [comment]
Other forks haven't tried to block the game's input because they have other interestest, while this one is focused on offering tools for making screenshots and videos on games. So if you are not doing anything like that then yes I suggest using other UnityExplorer forks.

**Comment by holetoo233** [comment]
> Other forks haven't tried to block the game's input because they have other interestest, while this one is focused on offering tools for making screenshots and videos on games. So if you are not doing anything like that then yes I suggest using other UnityExplorer forks.

ok，thanks
